source/frontend/src/interfaces/IProjectSearchResults.ts

```diff
@@ -0,0 +1,9 @@
+export interface IProjectSearchResult {
```

Comment:
same as the backend. This could simply be the projectModel. Also, models from the api start with "API_*"

Reply:
Yes interfaces starting with I are frontend use only interfaces (not api related)